pkgs/os-specific/linux/kernel/0001-KVM-external-hypervisor-for-imx8-5.10.72-kernel.patch

```diff
@@ -0,0 +1,2051 @@
+From e83aaa5788a779cae1e28f976a624647ee2fa0fa Mon Sep 17 00:00:00 2001
+From: Janne Karhunen <Janne.Karhunen@gmail.com>
+Date: Tue, 21 Apr 2020 09:45:23 +0300
+Subject: [PATCH] KVM: external hypervisor for imx8qmmek 5.10.72 kernel
```

Comment:
Does this apply to the newer `5.15.32` kernel we updated yesterday? https://github.com/tiiuae/nixpkgs-spectrum/commit/60ddbbe6ccdebd1ec7f8bbc60cf2c2ead0824248